lib/types/cql-utils.js

```diff
@@ -14,6 +14,9 @@ const Duration = require("./duration");
  * @returns {rust.QueryParameterWrapper}
  */
 function getWrapped(type, value) {
+    if (value === null || value === types.unset) {
+        return rust.QueryParameterWrapper.fromEmpty();
+    }
```

Comment:
`CqlValue::Empty` is something totally different than the DB concept of `null`. Please don't mix them up.